core/src/avm2/e4x.rs

```diff
@@ -774,7 +808,7 @@ pub fn to_xml_string<'gc>(
     activation: &mut Activation<'_, 'gc>,
 ) -> Result<AvmString<'gc>, Error<'gc>> {
```

Comment:
This can return an `AvmString<'gc>` now since it will always return `Ok`

Reply:
Fixed (and a few other methods)